snapcraft/_store.py

```diff
@@ -547,11 +547,8 @@ def _maybe_prompt_for_key(name):
 def register_key(name):
     key = _maybe_prompt_for_key(name)
     store_client = StoreClientCLI()
-    try:
-        if not login(store=store_client, acls=["modify_account_key"], save=False):
-            raise storeapi.errors.LoginRequiredError()
-    except storeapi.errors.StoreAuthenticationError as e:
-        raise storeapi.errors.LoginRequiredError(str(e)) from e
```

Comment:
is LoginRequiredError still needed or can it be removed?

Reply:
Removed! thnx